app/views/works/_additional_form.html.erb

```diff
@@ -19,10 +19,9 @@
     </select>
   </div>
 
-  <!-- ARK field  -->
-  <% if @work.ark.present? %>
-    <div class="field">
-      ARK: <%= @work.ark %>
-    </div>
-  <% end %>
+  <!-- ARK field (mostly for legacy DSpace works) -->
+  <div class="field">
+    ARK: <br>
+    <input type="text" id="ark" name="ark" class="input-text-long" value="<%= @work.resource.ark %>" />
```

Comment:
Notice that the ARK a field is now a user entered field (in the Additional Fields tab). This will allow us to indicate the ARK value for records that we migrate from DataSpace.